packages/api/src/schema/schema.yaml

```
@@ -658,6 +661,11 @@ components:
       required: []
       table: null
 
+    webhook-patch-payload:
+      $ref: "#/components/schemas/webhook"
+      required: []
+      table: null
```

This will allow requests with any webhook fields to go through, they will just silently fail because we ignore them. So I think it's a better experience for the API user if you explicitly list the editable fields here, like on the `stream-patch-payload` (and make sure to have a `additionalProperties: false`).

---

Yeah makes much more sense. Thanks for this suggestion and the different way to approach the update of the webhook, I am updating both